br-ext/configs/toolchain-aarch64-sdk

```diff
@@ -0,0 +1,9 @@
+BR2_TOOLCHAIN_EXTERNAL=y
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM=y
+BR2_TOOLCHAIN_EXTERNAL_PATH="%TOP_DIR%/toolchains/aarch64"
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM_PREFIX="aarch64-linux"
```

Comment:
Why not keep the traditional prefix, `aarch64-linux-gnu`?

Reply:
I don't know how to configure "make sdk" to do that. Here we're using what "make sdk" produced.

Reply:
Ah, that's fine then.